Neos.Neos/Classes/Eel/FlowQueryOperations/ParentsOperation.php

```diff
@@ -45,33 +46,38 @@ class ParentsOperation extends AbstractOperation
      */
     public function canEvaluate($context)
     {
-        return count($context) === 0 || (isset($context[0]) && ($context[0] instanceof NodeInterface) && ($context[0]->getContext() instanceof ContentContext));
+        return count($context) === 0 || (isset($context[0]) && ($context[0] instanceof TraversableNodeInterface));
     }
 
     /**
      * {@inheritdoc}
      *
      * @param FlowQuery $flowQuery the FlowQuery object
      * @param array $arguments the arguments for this operation
+     * @todo Compare to node type Neos.Neos:Site instead of path once it is available
```

Comment:
just wondering: What would be the advantage to comparing the node type over its path?

Reply:
The "path" is an expensive operation in the new CR